.taskcat.yml

```diff
@@ -5,102 +5,11 @@ project:
   lambda_zip_path: functions/packages
   s3_regional_buckets: true
 tests:
-  # full:
-  #   parameters:
-  #     QSS3BucketName: $[taskcat_autobucket]
-  #     QSS3BucketRegion: $[taskcat_current_region]
-  #   regions:
-  #     - af-south-1
-  #     - ap-northeast-1
-  #     - ap-northeast-2
-  #     - ap-south-1
-  #     - ap-east-1
-  #     - ap-southeast-1
-  #     - ap-southeast-2
-  #     - ca-central-1
-  #     - eu-central-1
-  #     - eu-north-1
-  #     - eu-west-1
-  #     - eu-west-2
-  #     - eu-west-3
-  #     - me-south-1
-  #     - sa-east-1
-  #     - us-east-1
-  #     - us-east-2
-  #     - us-west-1
-  #     - us-west-2
-  ecr-auto:
+  full-no-auto:
```

Comment:
The "auto" template requires a valid Snyk Group ID and API token, and we can't put these in the `.taskcat.yml`, so I opted to test the "full" template instead which doesn't require these parameters.